File: Programming Basic/Conditional_statements_exercise/01_Sum_Seconds.py
import math

time_first = float(input())
time_second = float(input())
time_third = float(input())

total_time = time_first + time_second + time_third

minutes = total_time / 60
seconds = total_time % 60

minutes = math.floor(minutes)
seconds = round(seconds)

if seconds < 10:
    print(f'{minutes}:0{seconds}')
else:
    print(f'{minutes}:{seconds}')

File: Java Script Basic - April 2020/Conditional_Statements_Exercise/01_Sum_Seconds.js
function sumSeconds(arg1, arg2, arg3) {
    let time_first = parseFloat(arg1)
    let time_second = parseFloat(arg2)
    let time_third = parseFloat(arg3)

    let total_time = time_first + time_second + time_third

    let minutes = Math.floor(total_time / 60)
    let seconds = Math.round(total_time % 60)

    if (seconds < 10) {
        console.log(`${minutes}:0${seconds}`)
    } else {
        console.log(`${minutes}:${seconds}`)
    }

}

sumSeconds(35,45,44)